sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/MessageUtilsTest.java

```diff
@@ -60,4 +68,33 @@ void convertUUIDToDotNetBytes() {
         // Assert
         assertArrayEquals(dotNetGuidBytes, convertedBytes, "UUID conversion from Java to DotNet failed");
     }
+
+    @ParameterizedTest
+    @MethodSource("calcTotalTimeoutTestData")
+    void calcTotalTimeout(List<Object> testData) {
```

Comment:
This is hard to read. You can pass in a Stream<Arguments> and it'll try to deserialize it into the actual typees. Look for uses in our other tests.

Reply:
Changed to use Arguments.of and auto-deserialize. Thanks for the suggestion.